docs/src/pages/[platform]/components/heading/demo.tsx

```diff
@@ -1,21 +1,44 @@
 import * as React from 'react';
-import { Heading, HeadingLevel } from '@aws-amplify/ui-react';
+import { Heading, HeadingLevel, HeadingProps } from '@aws-amplify/ui-react';
 
 import { HeadingPropControls } from './HeadingPropControls';
 import { Demo } from '@/components/Demo';
 
-const propsToCode = (level: HeadingLevel) => {
-  return `<Heading level={${level}}>Heading</Heading>`;
+const propsToCode = (
+  level: HeadingLevel,
+  isTruncated: boolean,
+  value: string
+) => {
+  return `<Heading
```

Comment:
nit: Can we fix the indentation here to match the other demos?
```jsx
<Heading
  width='30vw'
  level={${level}} 
  isTruncated={${isTruncated}}
>
  ${value}
</Heading>
```

Reply:
Done

Reply:
<img width="883" alt="image" src="https://user-images.githubusercontent.com/11983489/195157580-20e2e10b-2d69-4745-a663-7a4e353c30d9.png">